### DISCUSSION TITLE
Should I modifed?

### DISCUSSION THREAD

**Discussion by DinhHuy2010** [ThioJoe/YT-Spammer-Purge#715] [Q&A] [answered]
I have open this repo in github.dev. Should I modfied??? @ThioJoe 
![Screenshot 2022-03-04 094235](https://user-images.githubusercontent.com/95196459/156689802-f2af0106-0bf3-4d55-b164-0922726da08a.png)

**Comment by Masamune3210** [comment, marked as answer]
What are you asking? If you are wanting to know if you can alter things, alterations can be submitted as pull requests where they will be either merged or denied
